docs/bakefile.md

```
@@ -0,0 +1,80 @@
+- `--tar-path`
+- `--oci-layout-path`
+
+Name a single target to use them.
+
+## Relationship to docker-bake.hcl
+
+kaniko uses its own names. An image reference is a `destination`, not a `tag`. There is no per-target `context` or `dockerfile`, those are flags. Variables, functions, `group`, `inherits`, `matrix` and attestations are not supported.
+
+A buildx bakefile fails with the construct it does not understand named:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)